src/main/java/org/hpccsystems/jdbcdriver/HPCCDatabaseMetaData.java

```diff
@@ -32,15 +32,16 @@
 import java.util.logging.Level;
 
 import org.hpccsystems.jdbcdriver.HPCCJDBCUtils.EclTypes;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.HPCCColumn;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.HPCCQuerySet;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.HPCCTable;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.OutputDataset;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.PublishedQuery;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.QuerySetAliasMap;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.QuerySignature;
-import org.hpccsystems.ws.client.gen.wsdfu.v1_51.DFUDataColumn;
-import org.hpccsystems.ws.client.gen.wsdfu.v1_51.DFUSearchDataResponse;
+import org.hpccsystems.ws.client.gen.axis2.wsdfu.v1_56.DFUDataColumn;
```

Comment:
if it makes sense, please open jira in wsclient to provide wrapper based interface 
DFUDataColumnWrapper, DFUSearchDataResponseWrapper, ArrayOfDFUDataColumnWrapper